### PR TITLE
DOC-2513: In read-only mode the editor now allows normal cursor movement and block element selection, including video playback & New  property for all ui components.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2513 & DOC-2540 & DOC-2497

Site: [Context.adoc new page](http://docs-feature-74-doc-2513doc-2540.staging.tiny.cloud/docs/tinymce/latest/context/)
Site: [Release note entry 1](http://docs-feature-74-doc-2513doc-2540.staging.tiny.cloud/docs/tinymce/latest/7.4-release-notes/#in-read-only-mode-the-editor-now-allows-normal-cursor-movement-and-block-element-selection-including-video-playback)
Site: [Release note entry 2](http://docs-feature-74-doc-2513doc-2540.staging.tiny.cloud/docs/tinymce/latest/7.4-release-notes/#new-context-property-for-all-ui-components)

Changes:
* Added `In read-only mode the editor now allows normal cursor movement and block element selection, including video playback.`
* Added `New  property for all ui components.`
* Added new `context.adoc`.

- [x] TODO:// Update `state-updates` @shanmen-tiny 

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`.
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`.
- [x] Included a `release note` entry for any `New product features`.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/custom-split-toolbar-button.adoc [hunk 1]
```diff
@@ -19,6 +19,8 @@ include::partial$misc/admon-predefined-icons-only.adoc[]
 |fetch |`+(success: (menu) => void) => void+` |Drop-down menu |required |A callback function that should be passed a list of xref:choice-menu-items[choice menu items] for the dropdown menu.
 |onItemAction |`+(api, value) => void+` |Choice menu items |required |Function invoked when a dropdown list option is clicked. The `+value+` is passed from the selected choice menu item.
 |onSetup |`+(api) => (api) => void+` |All |optional |default: `+() => () => {}+` - Function invoked when the button is rendered. For details, see: xref:using-onsetup[Using `+onSetup+`].
+include::partial$misc/admon-requires-7.4v.adoc[]
+|context |string |optional |default: `mode:design` - The context property dynamically enables or disables the button based on the editor's current state. For details, see: xref:context.adoc[Context].
```
Comment (kemister85) [inline modules/ROOT/pages/custom-split-toolbar-button.adoc:23]: note: this is not rendering

**Review by shanmen-tiny — APPROVED** [review]

File: modules/ROOT/pages/dialog-components.adoc [hunk 2]
```diff
@@ -226,6 +226,8 @@ include::partial$misc/admon-predefined-icons-only.adoc[]
 *When configured, the button will display the icon instead of text.*
 |buttonType |`+'primary'+`, `+'secondary'+`, or `+'toolbar'+` |optional |default: `+'secondary'+` - Whether to style the button as a primary, secondary, or toolbar button.
 |borderless |boolean |optional |default: `+false+` - Whether to style the button without a border and background color.
+include::partial$misc/admon-requires-7.4v.adoc[]
```
Comment (kemister85) [inline modules/ROOT/pages/dialog-components.adoc:229]:
```suggestion
4+|
include::partial$misc/admon-requires-7.4v.adoc[]
```

File: modules/ROOT/pages/dialog-components.adoc [hunk 3]
```diff
@@ -399,6 +405,8 @@ An *input* is a composite component that renders a label and a single line text
 |placeholder |string |optional |String of placeholder text for the input field.
 |enabled |boolean |optional |Allows the field to be disabled. Default is `+true+`.
 |inputMode |string |optional |Allows for the specification of input type for displaying contextual onscreen keyboards on mobile devices.
+include::partial$misc/admon-requires-7.4v.adoc[]
```
Comment (kemister85) [inline modules/ROOT/pages/dialog-components.adoc:408]:
```suggestion
4+|
include::partial$misc/admon-requires-7.4v.adoc[]
```

File: modules/ROOT/pages/dialog-components.adoc [hunk 4]
```diff
@@ -724,6 +737,9 @@ If `picker_text` is not specified, the tooltip defaults to the value provided by
 
 
 This option *requires* `+file_picker_callback+` to be configured.
+
+include::partial$misc/admon-requires-7.4v.adoc[]
```
Comment (kemister85) [inline modules/ROOT/pages/dialog-components.adoc:741]:
```suggestion
4+|
include::partial$misc/admon-requires-7.4v.adoc[]
```

File: modules/ROOT/pages/dialog-footer-buttons.adoc [hunk 5]
```diff
@@ -21,6 +21,8 @@ include::partial$misc/admon-predefined-icons-only.adoc[]
 |buttonType |`+'primary'+`, or `+'secondary'+` |optional |default: `+'secondary'+` - Whether to style the button as a primary, or secondary button.
 |enabled |boolean |optional |default: `+true+` - When `+false+`, the button will be disabled when the dialog loads.
 |align |`+'end'+` or `+'start'+` |optional |default: `+'end'+` - When set to `+'end'+` the button will display on the right-hand side of the dialog. When set to `+'start'+` the button will display on the left-hand side.
+include::partial$misc/admon-requires-7.4v.adoc[]
```
Comment (kemister85) [inline modules/ROOT/pages/dialog-footer-buttons.adoc:24]:
```suggestion
4+|
include::partial$misc/admon-requires-7.4v.adoc[]
```

File: modules/ROOT/pages/dialog-footer-buttons.adoc [hunk 6]
```diff
@@ -89,6 +92,8 @@ The following options can be specified for a dialog menu button _item_:
 |type |string |required |The type `+togglemenuitem+` should be used.
 |text |string |optional |Text to display if no icon is found.
 |value |string |optional |A value to associate with the menu item.
+include::partial$misc/admon-requires-7.4v.adoc[]
```
Comment (kemister85) [inline modules/ROOT/pages/dialog-footer-buttons.adoc:95]:
```suggestion
4+|
include::partial$misc/admon-requires-7.4v.adoc[]
```

**Pull request merged**